scylla/src/transport/session.rs

```diff
@@ -1056,10 +1056,19 @@ impl Session {
     }
 
     pub async fn check_schema_agreement(&self) -> Result<bool, QueryError> {
-        self.schema_agreement_auxilary(|connection: Arc<Connection>| async move {
-            connection.check_schema_agreement().await
```

Comment:
Sounds like `connection::check_schema_agreement` is no longer used anywhere and can be removed, right? If so, let's do it in this patch

Reply:
I thought it could be useful in optimization, but after sleep I deemed it to be unnecessary. Removed.